code/datums/traits/_quirk.dm

```diff
@@ -65,6 +65,13 @@
 		return
 	if(quirk_holder.stat == DEAD)
 		return
+	// post_add is used to get settings from the client prefs
+	// validate that a client is attached before running
+	if(!post_add_ran)
+		var/mob/user = quirk_holder
+		if(user && user.client)
```

Comment:
`quirk_holder` is already typed as living and prior checks imply it exists at this point. Just do something like `if(!(quirk_flags & QUIRK_POST_ADD_RAN) && quirk_holder.client)`

Reply:
Fixed this up also.